src/components/apply/avatar.tsx

```diff
@@ -30,17 +31,18 @@ function getAvatarImage(avatar: AvatarType) {
 
 const Avatar = ({ avatar }: AvatarProps) => {
```

Comment:
nit: I know this wasn't in this pr, but could you make this a regular export? I don't like unnecessarily using default exports (unless they're required by next.js)

Reply:
tried removing default but would get Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Reply:
yeah you'd need to change the imports as well lol